web/v2/player/index.ts

```diff
@@ -124,7 +107,12 @@ export interface APIPlayerCompanyHistory {
  * Information about a TruckersMP player.
  * @see https://truckersmp.com/developers/api#operation/get-player-id
  */
-export interface APIPlayer {
+export type APIPlayer = _Base & _APIPlayerVTCHistory & _APIPlayerBanData;
+
+// #region APIPlayer
+
+/** Base interface containing property annotations */
```

Comment:
A little bit of nitpicking, though, can we stick to multiline comments to remain consistent, please?

The same goes for `web/v2/player/patreon.ts`.